rust/noosphere-ns/README.md

```diff
@@ -3,3 +3,12 @@
 # noosphere-ns
 
 Noosphere's P2P name system.
+
+## Bootstrap Node
+
+The `bootstrap_nns` binary target is an executable that runs one or many bootstrap
+nodes, based on configuration.
+
+```
+cargo run --bin bootstrap_nns -- run --key my-key-name --port 6666
```

Comment:
Any particular reason to use `_` instead of `-`? Most commands I know that are separated like this use `-`...

Reply:
:+1: aye `-` makes more sense and is consistent with pretty much everything in `/usr/bin/`